packages/react-core/src/components/Button/__snapshots__/Button.test.js.snap

```diff
@@ -14,19 +14,50 @@ exports[`action button 1`] = `
   background-color: transparent;
   border: 0px;
   border-radius: 30em;
+  text-decoration: none;
 }
 
 <button
-  aria-label=""
+  aria-disabled={null}
+  aria-label={null}
   className="pf-c-button pf-m-action"
   disabled={false}
+  tabIndex={null}
   type="button"
 >
   action
    Button
 </button>
 `;
 
+exports[`aria-disabled is set to true and tabIndex to -1 if component is not a btton and is disabled 1`] = `
+.pf-c-button.pf-m-primary.pf-m-disabled {
+  display: inline-block;
+  position: relative;
+  padding: 0.25rem 1.5rem 0.25rem 1.5rem;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #72767b;
+  text-align: center;
+  white-space: nowrap;
+  background-color: #d1d1d1;
+  border: 0px;
+  border-radius: 30em;
+  text-decoration: none;
+  pointer-events: none;
+}
+
+<a
+  aria-disabled={true}
+  aria-label={null}
+  className="pf-c-button pf-m-primary pf-m-disabled"
+  disabled={null}
+  tabIndex={-1}
+  type={null}
+/>
```

Comment:
Just flagging this as looking odd, compared to the `<button>` examples, which have both open and closing tags.

Reply:
This was due to the test not passing children in.  I went ahead and added children so it matches the other tests.